nansat/nansat.py

```diff
@@ -412,12 +414,13 @@ def resize(self, factor=None, width=None, height=None, pixelsize=None, resample_
             A factor is calculated from ratio of the
             current pixelsize to the desired pixelsize.
         resample_alg : int (GDALResampleAlg), optional
-               -1 : Average (default),
-                0 : NearestNeighbour
-                1 : Bilinear,
-                2 : Cubic,
-                3 : CubicSpline,
-                4 : Lancoz
+
+            - 1 : Average (default),
```

Comment:
Oops! -1 (minus one) is Average. Not - 1 (dash one). Should be : `- -1: Average (default)`